classifai.php

```diff
@@ -65,6 +65,7 @@ function classifai_define( $name, $value ) {
 }
 
 require_once __DIR__ . '/config.php';
+classifai_define( 'CLASSIFAI_PLUGIN_BASENAME', plugin_basename( __FILE__ ) );
```

Comment:
Feels like this belongs in the `config.php` file, unless there's a reason it can't be there

Reply:
Probably just an oddity with ClassifAI not including those bits in the classifai.php file whereas most our other plugins have them in the main plugin file 🤷🏼

Reply:
@dkotter I have kept this in the main plugin file due to `plugin_basename( __FILE__ ) )` because moving this to config.php will give us `config.php`  in plugin basename